### ISSUE TITLE
[suggestion] gzip manual pages when creating packages

### ISSUE THREAD

**Issue by cstrouse** [skycocker/chromebrew#511]
Since we have limited storage and a full eMMC drags it's feet I think we should make sure that manual pages are gzipped with the highest compression option. It's probably also a good idea to start stripping binaries when possible like other packagers tend to do.

Anyone disagree?

**Comment by jam7** [comment]
How about not installing static libraries?

**Comment by cstrouse** [comment]
That too is a good idea. We should create a checklist on the wiki for new package creation and package maintenance.

**Comment by jam7** [comment]
Recently, following @lyxell suggestion, I'm stripping binary and library.  It works great.  :)

`strip` works for binary.  8MB to 2MB.
```
-rwxr-xr-x chronos/chronos 8999144 2017-05-10 16:09 usr/local/bin/git
-rwxr-xr-x chronos/chronos 1831608 2017-05-11 01:07 usr/local/bin/git
```
`strip -S` works for library.  4MB to 1MB.
```
-rwxr-xr-x chronos/chronos 4553508 2017-05-10 06:39 usr/local/lib/libxml2.so.2.9.4
-rwxr-xr-x chronos/chronos 1299160 2017-05-10 21:16 usr/local/lib/libxml2.so.2.9.4
```

**Comment by uberhacker** [comment]
Resolved by #810?

**Comment by jam7** [comment]
Close in favor of #802 and #810.
